.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ aliases:
       - ~/.cache/yarn
   - &run_yarn_install
     name: Install dependencies
-    command: yarn install --frozen-lockfile
+    command: yarn install --frozen-lockfile --ignore-scripts
```

Comment:
This was the problem. Without --ignore-scripts, yarn would run the prepare script, which would trigger a whole build and timeout the install process.